pithos/pithos.py

```diff
@@ -520,6 +520,7 @@ def user_pause(self, *ignore):
 
     def pause(self):
         self.playing = False
+        self.destroy_ui_loop()
```

Comment:
If the loop is not manually destroyed if you click play more than once in 1 sec a new loop is created  before the old loop dies resulting in multiple loops. The whole point of the patch is to prevent that.